Libs/MRML/Widgets/qMRMLSliceControllerWidget.h

```diff
@@ -307,6 +313,15 @@ public slots:
   qMRMLSliceControllerWidget(qMRMLSliceControllerWidgetPrivate* obj,
                              QWidget* parent);
 
+  /// Value for slab spacing
+  /// Taken from the script repository
+  /// https://slicer.readthedocs.io/en/latest/developer_guide/script_repository/
+  double slabSpacing = 0.3;
```

Comment:
I'm not sure if this can be hardcoded. We definitely should not have any spacing value hardcoded in physical units. We could have some kind of an oversampling factor, which would tell if we aim for higher-quality (and slower) projection or lower-quality (and faster) computation. That should be named something like "slabOversamplingFactor" with a default of  2.0 (which would mean that we sample the volume at spacing/2.0 distance). This may be a constant for now, but it has to be specified in a logic class (probably in the slice logic).